WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -295,9 +295,8 @@ - (void)configurePublicizeCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath
     }
 
     // Configure the image
-    UIImage *image = [WPStyleGuide iconForService: publicizer.serviceID];
+    UIImage *image = [WPStyleGuide socialIconFor:publicizer.serviceID];
     [cell.imageView setImage:image];
-    cell.imageView.tintColor = ([connections count] > 0) ? [WPStyleGuide tintColorForConnectedService: publicizer.serviceID] : [UIColor murielListIcon];
```

Comment:
Please remove unused tintColorForConnectedService.

Reply:
It's still used in `PostSettingsViewController`. Let's make sure this change doesn't affect this screen and remove it in this screen to if possible. If not, it's good to merge 👍

Reply:
Will do this in a different PR when we remove the remote feature flag.